jdaviz/core/validunits.py

```diff
@@ -73,12 +73,12 @@ def create_flux_equivalencies_list(flux_unit, spectral_axis_unit):
 
     # Get local flux units.
     locally_defined_flux_units = ['Jy', 'mJy', 'uJy', 'MJy',
-                                  'W / (Hz m2)',
-                                  'eV / (s m2 Hz)',
-                                  'erg / (s cm2 Hz)',
-                                  'erg / (s cm2 Angstrom)',
+                                  'W / (m2 Hz)', 'W / (Hz m2)',  # Order is different in astropy v5.3  # noqa
+                                  'eV / (s m2 Hz)', 'eV / (Hz s m2)',
+                                  'erg / (s cm2 Angstrom)', 'erg / (s cm2 Angstrom)',
```

Comment:
These two are supposed to have the denominator units in the opposite order, right?

Reply:
I have gone through and removed the change in order (the only relevance having multiple orders would be from the API, but if the user looks at the flux_unit.choices they should see the correct order the string should be typed in).